LockboxXCUITests/LockboxXCUITests.swift

```diff
@@ -219,21 +182,36 @@ class LockboxXCUITests: BaseTestCase {
         } else {
             XCTAssertEqual(noMatches, 1)
         }
-
         // Tap on cacel
         app.buttons["Cancel"].tap()
         let searchFieldValueAfterCancel = searchTextField.placeholderValue
         XCTAssertEqual(searchFieldValueAfterCancel, "Search your entries")
-
         // Tap on 'x'
         searchTextField.tap()
         searchTextField.typeText("a")
         app.buttons["Clear text"].tap()
         let searchFieldValueAfterXButton = searchTextField.value as! String
         XCTAssertEqual(searchFieldValueAfterXButton, "Search your entries")
+        app.buttons["Cancel"].tap()
+        navigator.nowAt(Screen.LockboxMainPage)
     }
 
-    func test9LockNowUnlock() {
+    func testCheckAutolock() {
+        loginFxAccount()
+        // Check if the account is verified and if not, verify it
+        skipAutofillConfiguration()
+        tapOnFinishButton()
+        waitForLockboxEntriesListView()
```

Comment:
would it be a good idea to unify these login methods into one `loginToEntryListView` or something similar? it seems like they are repeated for most of the tests.